react/Checkbox/Checkbox.js

```diff
@@ -22,10 +22,10 @@ export default class Checkbox extends Component {
 
   static propTypes = {
     id: PropTypes.string.isRequired,
-    label: PropTypes.string.isRequired,
+    label: PropTypes.node.isRequired,
     className: PropTypes.string,
     inputProps: PropTypes.shape({
-      onChange: PropTypes.func.isRequired,
```

Comment:
That's not something we currently support though... just trying to make sure we don't lose any prop type checks here.  I'm happy to remove it because I'm not sure there's much value of enforcing it that prop